packages/text-annotator-react/src/TextAnnotator.tsx

```diff
@@ -1,13 +1,16 @@
 import { ReactNode, useContext, useEffect, useRef } from 'react';
 import { AnnotoriousContext, DrawingStyle, Filter } from '@annotorious/react';
+import type { FormatAdapter } from '@annotorious/core';
 import type { TextAnnotation, TextAnnotatorOptions } from '@recogito/text-annotator';
 import { createTextAnnotator } from '@recogito/text-annotator';
 
 import '@recogito/text-annotator/dist/text-annotator.css';
 
-export type TextAnnotatorProps<E extends unknown> = TextAnnotatorOptions<E> & {
+export interface TextAnnotatorProps<E extends unknown> extends Omit<TextAnnotatorOptions<E>, 'adapter'>  {
```

Comment:
W/o using the `interfaced`  Typescript still points to the `adapter` prop from the `TextAnnotatorOptions`, not the overridden one